public/apps/configuration/panels/tenant-list/configure_tab1.tsx

```diff
@@ -328,7 +328,7 @@ export function ConfigureTab1(props: AppDependencies) {
         <EuiPageContentHeader>
           <EuiPageContentHeaderSection>
             <EuiTitle size="l">
-              <h3>Multi-tenancy</h3>
+              <h3>Dashboards multi-tenancy</h3>
```

Comment:
Is the language `Dashboards ..` approved by UX? To me the word "Dashboards" seems repetitive and not necessary, but I will leave the decision up to UI/UX